TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/ColorDetect.java

```diff
@@ -0,0 +1,52 @@
+package org.firstinspires.ftc.teamcode.utilities;
+
+import static org.firstinspires.ftc.robotcore.external.BlocksOpModeCompanion.telemetry;
+
+import android.graphics.Color;
+
+import com.acmerobotics.dashboard.config.Config;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+@Config
+public class ColorDetect {
+    ColorSensor colorSensor;
+
+    public ColorDetect(HardwareMap hMap) {
+        this.colorSensor = hMap.get(ColorSensor.class, "color");
```

Comment:
probably should be adding this to config file

Reply:
i do not know what that means

Reply:
you need to add ports for all hardware objects in the file called CONFIG.java